modules/custom/social_magic_login/src/Service/MagicUrl.php

```diff
@@ -43,7 +43,7 @@ public function create(UserInterface $account, string $destination, array $optio
 
     // Get url options and prerequisites.
     $timestamp = \Drupal::time()->getRequestTime();
-    $lang_code = isset($options['langcode']) ? $options['langcode'] : $account->getPreferredLangcode();
+    $lang_code = $options['langcode'] ?? $account->getPreferredLangcode();
```

Comment:
```suggestion
    $lang_code = isset($options['langcode']) ? $options['langcode'] : $account->getPreferredLangcode();
```

This is PHP 7.0 code that we cannot support yet in the `5.x` branch.
When we're using Drupal 8.7.0 we can start using PHP 7.0 code.